src/apps/amp/ethereum/amp.farm.contract-position-fetcher.ts

```diff
@@ -45,14 +38,39 @@ export class EthereumAmpFarmContractPositionFetcher extends SingleStakingFarmTem
     key: (address: string) => `apps-v3:balance:ethereum:amp:api-data:${address}`,
     ttl: 5 * 60, // 5 minutes
   })
-  async getApiData(address: string) {
+  async getDefinitions(): Promise<DefaultContractPositionDefinition[]> {
+    return [{ address: '0x706d7f8b3445d8dfc790c524e3990ef014e7c578' }];
+  }
+
+  async getTokenDefinitions() {
+    return [
+      {
+        metaType: MetaType.SUPPLIED,
+        address: '0xff20817765cb7f73d4bde2e66e067e58d11095c2',
+        network: this.network,
+      },
+      //add second position to represent earned rewards that are not claimable and auto compounded
+      //we do not want claimable label on front end
+      {
+        metaType: MetaType.SUPPLIED,
```

Comment:
If you want to show earned rewards (auto compounded), you will have to chose a different metaType, otherwise it won't show on Zapper FE even thought the balances are returned when testing locally

Reply:
Supplied is the desired front end outcome as it is a clean row with no label next to the token ticker. Claimable adds the green claimable label to the right of the token ticker. Logging MetaType, these are the options available:
{
  WALLET: 'wallet',
  SUPPLIED: 'supplied',
  BORROWED: 'borrowed',
  CLAIMABLE: 'claimable',
  VESTING: 'vesting',
  LOCKED: 'locked',
  NFT: 'nft'
}
Do any of those have a similar front end look as Supplied? Claimable is good because it stays in the same grouping as supplied, but the green claimable label is not what we want. However if that is the option, we will use it.

Reply:
I would suggest you used `LOCKED`. I know that it's not locked, but it's kinda the most accurate one and there's no pill on the FE. By "pill" I mean the green claimable label or the yellow debt label

Reply:
Thanks Will!